src/main/java/frc/robot/RobotContainer.java

```diff
@@ -116,7 +116,7 @@ public RobotContainer() {
     }
 
     configureBindings();
-    // shooterSysIdBindings();
+    shooterSysIdBindings();
```

Comment:
We don't want to enable sysIdBindings in baseline.  This should be commented out.